src/components/ProjectCard.tsx

```diff
@@ -169,11 +173,18 @@ export default function ProjectCard({
               </span>
             </div>
 
-            {metadata?.description && (
+            {tags?.length ? (
```

Comment:
on project cards, we'll show the tags if they've been defined. otherwise we'll show the description

reasoning here is that if we're only able to show the first few words of a description, tags would be more useful to the user